int64 a
int64 b
---
int64 pos_idx
